### ISSUE TITLE
Status: 405 Method Not Allowed in balde-0.1.2 release

### ISSUE THREAD

**Issue by menangen** [balde/balde#17]
After run examples: ./hello --runserver I have no \* Running on http://127.0.0.1:8080/ (threads: 10)

```
menan@ubuntu-server:~/c/balde/examples$ **./hello** --runserver
Status: 405 Method Not Allowed
Content-Type: text/plain; charset=utf-8
Content-Length: 73

405 Method Not Allowed
```

and app exit

On Ubuntu 16.04 x64

libglib2.0-dev: **2.48.0-1ubuntu4**
libfcgi-dev: **2.4.0-8.3**

**Comment by komputronika** [comment]
I also got the same problem. Waiting for the solution.

On Ubuntu x64.

**Comment by fabiosvm** [comment]
I'm having the same problem on Mac OS.

**Comment by rafaelmartins** [comment]
The runserver implementation is only available in the git repo for now, 0.1.2 don't have it
